src/seq/bioseq.jl

```diff
@@ -1027,3 +1027,80 @@ end
 function enc64{A}(::BioSequence{A}, x)
     return UInt64(encode(A, convert(eltype(A), x)))
 end
+
+
+# Sequences to Matrix
+# -------------------
+
+"""
+    seqmatrix{A<:Alphabet}(vseq::Vector{BioSequence{A}}, major = :seq|:site)
+
+Construct a matrix of nucleotides or amino acids from a BioSequence.
+
+If parameter major is set to `:site`, the matrix is created such that one
+nucleotide from each sequence is placed in each column i.e. the matrix is laid
+out in site-major order.
+This means that iteration over one position of many sequences is efficient,
+as julia arrays are laid out in column major order.
+
+If the parameter major is set to `:seq`, the matrix is created such that each
+sequence is placed in one column i.e. the matrix is laid out in sequence-major
+order.
+This means that iteration over each sequence is efficient,
+as julia arrays are laid out in column major order.
+
+# Examples
+```julia
+julia> seqs = [dna"AAAAAAAAAA", dna"TTTTTTTTTT", dna"CCCCCCCCCC", dna"GGGGGGGGGG"]
```

Comment:
Ok fixed to length = 3 (a few seqs are slightly longer than length 3, but that's intentional to test the truncating behaviour of the function - similar to how `mismatches` truncates).